brainlessbuild.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
+
+git submodule init
+git submodule update
+
+(
+	cd dependencies/physfs
```

Comment:
We should no longer need a 'system' version of physfs, the cmake build system should build and link our patched version. If we still end up using the system version, that's a bug and needs to be fixed.